### ISSUE TITLE
UI Libraries

### ISSUE THREAD

**Issue by Folcon** [JetBrains/skija#127]
Just wondering if there's been any addition to the UI libraries that you mentioned in your announcement post or any instructions around how this was done:
`For example, we’ve played with implementing java.awt.Graphics2D and running Swing on top of it—seems to work fine.`

Being able to work with a library like this while still being able to write reasonable UI controls would be amazing.

**Comment by tonsky** [comment]
No, not yet. Window management library is coming alogn, though: https://github.com/jetbrains/jwm

As for Swing example, it’s here: https://github.com/JetBrains/skija/blob/master/examples/scenes/src/SwingScene.java

Also, David Gilbert works on Graphics2D implementation that could power Swing in the future too (it’s not his primary focus, but might be a side-effect of his work) https://github.com/jfree/skijagraphics2d
